scalableminds/webknossos
# Remove several unused JS dependencies

While debugging the library version conflicts for PR #7505 I ran a check for unused libs with `depcheck`. While it flagged many false positive libs I think it correctly identified these 5 libs as unused:

#### *[Name]*: *[PR where the lib was introduced]*
- `ndarray-convolve`: [Quick Select Tool PR](https://github.com/scalableminds/webknossos/pull/6542)
- `react-sortable-tree-theme-file-explore`:    [Dataset Folder PR](https://github.com/scalableminds/webknossos/pull/6591)
- `strip-ansi`: [Virtual VX Logs](https://github.com/scalableminds/webknossos/pull/6579)
- `@use-it/interval`: [Recent Datasets conversion on Dashboard PR](https://github.com/scalableminds/webknossos/pull/5597)
- `cross-fetch`: [PR to Fix airbrake-js](https://github.com/scalableminds/webknossos/pull/3859)

Hopefully this decreases our bundle size a little bit and helps to avoid version conflicts in the future.

Command to detect unused libs.
```
npx depcheck . --specials=eslint,webpack,prettier,typescript,less,esbuild
```

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Delete `node_modules` and `yarn install`
- I did tests to verify that:
  - That throwing an Error will trigger a Airbrake notifications
  - That quick select works in AI and non-AI mode
  - That VX logs can still be viewed
  - That skeletons & segment tree components looked "normal"

### Issues:
- None

------
(Please delete unneeded items, merge only when none are left open)
